routes/users.js

```diff
@@ -891,16 +899,17 @@ router.delete_user = async function(user, action_owner_id){
         await Promise.all(plugins_info.map(function(plugin_info){
             return plugin_call(plugin_info, user.uid, user, action_owner_id);
         }));
-        await utils.freeUserId(user.uidnumber);
-        if(notif.mailSet()) {
-            // eslint-disable-next-line no-unused-vars
-            await notif.sendUser(mailOptions);
-        }
-        return true;
     }
+
+    await utils.freeUserId(user.uidnumber);
+    if(notif.mailSet()) {
+        // eslint-disable-next-line no-unused-vars
+        await notif.sendUser(mailOptions);
+    }
+    return true;
 };
 
-router.delete('/user/:id', async function(req, res){
+router.delete('/user/:id/message/:message', async function(req, res){
```

Comment:
Using a different route may brake existing api calls

Reply:
I think api is only call by frontend, maybe you have some other tools using this api ?

Reply:
not for now I think, but I really do not like to get a message in request , a forcing a message info for a DELETE operation